ruby/vampires.rb

```diff
@@ -0,0 +1,129 @@
+# DBC 4.4
+# Interview with the Vampire
+
+# Ask the user how many employees will be processed.
+# After the result is printed for the first employee, start over and repeat
+# until all the employees have been processed. 
+
+puts "How many employees are you processing today?"
+employees = gets.chomp
+
+number = 0
+while number < employees.to_i 
+  number += 1
+
+  # Ask a new employee the following questions: 
+  # What is your name? 
+  # How old are you? 
+  # What year were you born? 
+  # Our company cafeteria serves garlic bread. Would you like us to order some for you? 
+  # Would you like to enroll in the company's health insurance?
+  
+  puts "What is your name?"
+  name = gets.chomp
+  
+    # Do they have a vampire name??
+
+    if name == "Drake Cula"
+      vampire_name = true
+    elsif name == "Tu Fang"
+      vampire_name = true
+    else 
+      normal_name = true
+    end 
+  
+  puts "How old are you?"
+  alleged_age = gets.chomp
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp
+  
+  puts "Our cafeteria serves garlic bread. Would you like us to order some for you? (Y/N)"
+  garlic_bread = gets.chomp
+  
+  puts "Would you like to enroll in the company's health insurance? (Y/N)"
+  health_insurance = gets.chomp
+
+  # Add a loop to ask about allergies; if sunshine is an allergy, print "Probably a vampire."
+
+  puts "Do you have any allergies? List one at a time, then enter 'done' to exit."
+
+    allergy = ""
+    while allergy != "done"
+      allergy = gets.chomp
+      next unless allergy == "sunshine"
```

Comment:
Using `next` here is a bit confusing. Are you actually trying to reach the next iteration? It seems more like you should be controlling the loop using `while allergy != "done" || allergy != "sunshine"` 

Then you can handle the values that you gathered with the rest of the inputs to be processed.